lib/scsi_defs.h

```diff
@@ -225,9 +225,9 @@ struct InquiryData {
   bool reserved_6 : 1;
   bool cmdque : 1;  // Command Management Model bit
   bool vs_2 : 1;    // vendor specific bit
-  uint64_t vendor_identification : 64;
+  char vendor_identification[8];
   uint8_t product_identification[16];
-  uint32_t product_revision_level : 32;
+  uint8_t product_revision_level[4];
```

Comment:
What's the motivation for this change?

Reply:
This entry deals with ascii characters. Would be easier to work with in uint8_t's.

>   Shall be set to the last 4 ASCII graphic characters in the range of 21h-7Eh
>   (i.e. last 4 non-space characters) of the Firmware Revision (FR) field within
>   the Identify Controller Data Structure.